extension.json

```diff
@@ -14,6 +14,9 @@
   "schemas": [
     "release-schema.json"
   ],
+  "dependencies": [
+    "https://raw.githubusercontent.com/open-contracting-extensions/ocds_lots_extension/master/extension.json"
+  ],
```

Comment:
I think we do not need this dependency.

Reply:
without this dependency the tests return AssertionErrors as Lot is undefined

Reply:
Aha, put it under `testDependencies` not `dependencies`.

Reply:
I've now added a couple bullets here: https://ocds-standard-development-handbook.readthedocs.io/en/latest/extensions/index.html

I didn't add this to https://github.com/open-contracting/standard_extension_template as `testDependencies` has no utility for third-party extensions.

Reply:
ah! okay